tests/storage/storage.go

```diff
@@ -82,10 +82,10 @@ const (
 	// Without cloud init user data Cirros takes long time to boot,
 	// so provide this dummy data to make it boot faster
 	cirrosUserData = "#!/bin/bash\necho 'hello'\n"
-)
 
-const (
-	diskSerial = "FB-fb_18030C10002032"
+	defaultDiskSize  = "1Gi"
+	diskSerial       = "FB-fb_18030C10002032"
+	osAlpineHostPath = "alpine-host-path"
```

Comment:
Only commet I have is if it is necessary to declare the constant or we can simply hardcode the values in the function call. 
I don't have an opinion, it only removes 2 lines and avoid to scroll to look for the values

Reply:
I changed the consts as suggested. I like it better this way too.